docs/usage/guide_f5.rst

```diff
@@ -0,0 +1,316 @@
+:orphan: true
+
+F5 BIG-IP Guide
+===============
+
+BIG-IP systems can inspect, secure, balance, and accelerate the traffic that passes through your network.
+
+F5 provides modules for Ansible that you can use to deploy, provision, configure, and orchestrate BIG-IP systems. BIG-IP can be physical hardware or a virtual edition (BIG-IP VE) running in a public or private cloud.
+
+About the API
+`````````````
+
+The primary API that's included with BIG-IP is called iControl REST. The Python library that interacts with iControl REST is called ``f5-sdk``.
+
+To work with the F5 Modules for Ansible, you should install ``f5-sdk``, as well as:
+
+- ``bigsuds``, the F5 SOAP API that was used for modules prior to Ansible 2.2
+- ``netaddr``, which helps with address manipulation
+- ``deepdiff``, to compare and find the differences between objects
```

Comment:
I dont think this is a dependency any longer. Though it may still be listed as one in some docs. I'll remove it.

Reply:
Are you referring to just bigsuds, or are you referring to all three?

Reply:
@selhorn only deepdiff